src/tasks/publish/index.ts

```diff
@@ -249,3 +244,35 @@ async function _getChainId(bre: BuidlerRuntimeEnvironment): Promise<number> {
   const net = await provider.getNetwork()
   return net.chainId
 }
+
+/**
+ * Utility to parse appName from the aragon config
+ * @param appNameOrObj
+ * @param network
+ */
+function _parseAppName(
```

Comment:
What you think of moving this to the `appName` util file?

Reply:
Let's do!

Reply:
After looking at this, this a utility specific to the publish task need to read the aragon config. So I kept the utility there but change the name to be less confusing. 

Fixed with https://github.com/aragon/buidler-aragon/pull/89/commits/7ccebcb274cb752e041356316c53440d791410f0